re_data/command_line.py

```diff
@@ -197,4 +212,67 @@ def serve():
         httpd.server_close()
 
 
+@notify.command()
+@click.option(
+    '--start-date',
+    type=click.DateTime(formats=["%Y-%m-%d"]),
+    default=str((date.today() - timedelta(days=7)).strftime("%Y-%m-%d")),
+    help="Specify starting date to generate alert data, by default re_data will use 7 days ago for that value"
+)
+@click.option(
+    '--end-date',
+    type=click.DateTime(formats=["%Y-%m-%d"]),
+    default=str(date.today().strftime("%Y-%m-%d")),
+    help="""
+        Specify end date used in generating alert data, by default re_data will use current date for that.
+    """
+)
+@click.option(
+    '--webhook-url',
+    type=click.STRING,
+    required=True,
+    help="Incoming webhook url to post messages from external sources into Slack."
+         " e.g. https://hooks.slack.com/services/T0JKJQKQS/B0JKJQKQS/XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+)
+@click.option(
+    '--subtitle',
+    type=click.STRING,
+    default='',
+    help="Extra markdown text to be added to the alert message"
+)
+def slack(start_date, end_date, webhook_url, subtitle):
+    start_date = str(start_date.date())
+    end_date = str(end_date.date())
+    args = {
+        'start_date': start_date,
+        'end_date': end_date,
+    }
+    command_list = ['dbt', 'run-operation', 'export_alerts', '--args', yaml.dump(args)]
+    completed_process = subprocess.run(command_list)
+    completed_process.check_returncode()
+
+    re_data_dir = os.path.join(os.getcwd(), 'target', 're_data')
+    with open(os.path.join(re_data_dir, 'alerts.json')) as f:
+        alerts = json.load(f)
+    if len(alerts) > 0:
+        tabulated_alerts = format_alerts_to_table(alerts[:20])
+        message = f"""
+:red_circle: {len(alerts)} alerts found between {start_date} and {end_date}.
+{subtitle}
+
+_Showing most recent 20 alerts._
+<https://re-data.github.io/re-data/latest/docs/reference/cli/overview|Generate Observability UI> to show more details.
+
```

Comment:
@mateuszklimek i think we can change the link to `docs.getre.io` here before merging. what do you think?